test/Constraints/optional.swift

```diff
@@ -459,3 +459,27 @@ func sr_12309() {
     return (((nil))) // Ok
   }
 }
+
+// rdar://75146811 - crash due to incrrect inout type
```

Comment:
Just note on a minor typo `incrrect ` -> `incorrect` :)

Reply:
Ah, thank you! I'm going to clean that up in NFC PR.

Reply:
https://github.com/apple/swift/pull/36522